easybuild/easyconfigs/m/matplotlib/Qhull-intel-fix.patch

```diff
@@ -0,0 +1,22 @@
+# they use some weird template thing in pure C code to avoid
```

Comment:
shouldn't we version this patch file, i.e. `matplotlib-1.4.3_Qhull-intel-fix.patch`?